Manager.py

```diff
@@ -18,8 +18,14 @@ def resolve_status(runner,reader,outfiles):
     if not os.path.exists(outfile):
       return 'not_started'
 
-  #We are in an error state or we haven't collected 
-  #the results. 
+  
+  #We are in an error state 
+  for outf in outfiles:
+    lines = open(outf, 'r').read().split()
+    if ('HF_done' in  lines) and ('All_done' not in lines):
```

Comment:
This is specific to pyscf for a general function.

Looking for signals should be in the Reader object.